sechub-pds-solutions/multi/docker/pds-config.json

```diff
@@ -0,0 +1,79 @@
+{
```

Comment:
At first I did not get the reason "why a multi PDS solution" but I think now I got it...

Very good. We only maintain ONE PDS solution but can use different tools at the same time - even when I had the idea about multiple product by one PDS (i wrote the code...) I did not think about it as a production variant!

@Jeeppler : Very good idea! :rocket:  Will save us maintenance and is also easier to deploy...

Reply:
@de-jcup the name `pds-multi` is maybe not the best, but it is short. I wanted to show, that it is possible to have multiple products in a single PDS.